shared/css/color.scss

```diff
@@ -59,6 +62,7 @@ $hdr_color:   $purple;
 // Some extra colors.
 $red: #c00;
 $lightest_red: #fcc;
+$redesign_red: #d5001a;
```

Comment:
Removed the redesign_ colors and updated the screenshots.  They're really similar, except for the green.